app/models/role.rb

```diff
@@ -18,7 +18,7 @@
 class Role < ApplicationRecord
   # The roles that are referenced directly in the code.
   # Changing the name would break the website, so this list is used to prevent name changes for these roles.
-  HARDCODED_NAMES = ['Admin', 'Member', 'Committee', 'DM Trained', 'Business Manager'].freeze
```

Comment:
while you're doing this, can you add alcohol trained?

Reply:
call it bar trained actually

Reply:
Done